share/spack/templates/container/Dockerfile

```
@@ -1,6 +1,13 @@
 # Build stage with Spack pre-installed and ready to be used
 FROM {{ build.image }}:{{ build.tag }} as builder
 
+{% if os_packages_build %}
+# Install OS packages needed to build the software
+RUN {{ os_packages_build.update }}                                   \
+ && {{ os_packages_build.install }}{% for pkg in os_packages_build.list %} {{ pkg }}{% endfor %} \
```

What about this? Seems to work nicely for me.

```suggestion
RUN {{ os_packages_build.update }} \
 && {{ os_packages_build.install }} {{ os_packages_build.list | join | replace('\n', ' ') }} \
```